Sources/GenIR/XcodeLogParser.swift

```diff
@@ -67,105 +67,138 @@ class XcodeLogParser {
 		}
 	}
 
-	/// Parses an array representing the contents of an Xcode build log
-	/// - Parameters:
-	///   - lines: contents of the Xcode build log lines
-	private func parseBuildLog(_ lines: [String]) {
-		var currentTarget: String?
+	/// Parse the lines from the build log
+	func parseBuildLog() {
 		var seenTargets = Set<String>()
 
-		for (index, line) in lines.enumerated() {
-			let line = line.trimmingCharacters(in: .whitespacesAndNewlines)
+		while let line = consumeLine() {
+			if line.hasPrefix("Build description path: ") {
+				buildCachePath = buildDescriptionPath(from: line)
+			} else if line.hasPrefix("Build settings from command line:") {
+				settings = parseBuildSettings()
+			} else {
+				// Attempt to find a build task on this line that we are interested in.
+				let task = line.split(separator: " ", maxSplits: 1, omittingEmptySubsequences: false)[0]
+
+				switch task {
```

Comment:
Turns out the SwiftLint job was silently failing - I've updated it here: https://github.com/veracode/gen-ir/pull/67 we should merge that and rerun the pipelines (which will fail on this switch case)